pajbot/modules/basic/twitter.py

```diff
@@ -0,0 +1,85 @@
+import logging
+
+from pajbot.models.command import Command, CommandExample
+from pajbot.modules import BaseModule, ModuleSetting
+from pajbot.modules.basic import BasicCommandsModule
+from pajbot.managers.adminlog import AdminLogManager
+
+log = logging.getLogger(__name__)
+
+
+class TwitterModule(BaseModule):
+    ID = __name__.split(".")[-1]
+    NAME = "Twitter"
+    DESCRIPTION = "Enable !twitterfollow/!twitterunfollow command usage"
+    CATEGORY = "Feature"
+    ENABLED_DEFAULT = True
+    PARENT_MODULE = BasicCommandsModule
+    SETTINGS = [
+        ModuleSetting(
+            key="level",
+            label="Minimum level required to use commands",
+            type="number",
+            required=True,
+            placeholder="",
+            default=1000,
+            constraints={"min_value": 1000},
+        ),
+    ]
+
+    @staticmethod
+    def twitter_follow(bot, source, message, event, args):
+        if message:
+            username = message.split(" ")[0].strip().lower()
+            if bot.twitter_manager.follow_user(username):
+                log_msg = f"Now following {username}"
+                bot.whisper(source, log_msg)
+                AdminLogManager.add_entry("Twitter user followed", source, log_msg)
```

Comment:
There's not much context to this, has anyone got any suggestions for better wording?